playbooks/common/openshift-node/config.yml

```diff
@@ -27,10 +27,12 @@
     stat:
       path: "{{ item }}"
     with_items:
-    - "/etc/openshift/node/node.key"
-    - "/etc/openshift/node/node.kubeconfig"
+    - "/etc/openshift/node/system:node:{{ openshift.common.hostname }}.crt"
+    - "/etc/openshift/node/system:node:{{ openshift.common.hostname }}.key"
+    - "/etc/openshift/node/system:node:{{ openshift.common.hostname }}.kubeconfig"
```

Comment:
These paths previously matched the output of create-node-config. They now match the output of create-api-client-config, which just uses the passed username as the base name of the key/crt/kubeconfig files. If this was working on Friday, then it seems like something in the install scripts switched from calling `create-node-config` to calling `create-api-client-config`